tsv/latest/AnnotText.tsv

```diff
@@ -22,7 +22,7 @@ T	string-text	1.1		FALSE	FALSE	FALSE
 Popup	dictionary	1.3		FALSE	TRUE	FALSE				[AnnotPopup]	
 RC	stream;string-text	1.5		FALSE	[TRUE];[FALSE]	FALSE				[Stream];[]	
 CreationDate	date	1.5		FALSE	FALSE	FALSE					
-IRT	dictionary	1.5		fn:IsRequired(fn:IsPresent(RT))	FALSE	FALSE				[AnnotText,AnnotLink,AnnotFreeText,AnnotLine,AnnotSquare,AnnotCircle,AnnotPolygon,AnnotHighlight,AnnotUnderline,AnnotSquiggly,AnnotStrikeOut,AnnotCaret,AnnotStamp,AnnotInk,AnnotPopup,AnnotFileAttachment,AnnotSound,AnnotMovie,AnnotScreen,AnnotWidget,AnnotPrinterMark,fn:SinceVersion(1.6,AnnotWatermark),fn:SinceVersion(1.6,Annot3D),fn:SinceVersion(1.7,AnnotRedact),fn:SinceVersion(2.0,AnnotProjection),fn:SinceVersion(2.0,AnnotRichMedia)]	
+IRT	dictionary	1.5		fn:IsRequired(1.5,fn:IsPDFVersion(fn:IsPresent(State)) || fn:SinceVersion(1.6,fn:IsPresent(RT)))	FALSE	FALSE				[AnnotText,AnnotLink,AnnotFreeText,AnnotLine,AnnotSquare,AnnotCircle,AnnotPolygon,AnnotHighlight,AnnotUnderline,AnnotSquiggly,AnnotStrikeOut,AnnotCaret,AnnotStamp,AnnotInk,AnnotPopup,AnnotFileAttachment,AnnotSound,AnnotMovie,AnnotScreen,AnnotWidget,AnnotPrinterMark,fn:SinceVersion(1.6,AnnotWatermark),fn:SinceVersion(1.6,Annot3D),fn:SinceVersion(1.7,AnnotRedact),fn:SinceVersion(2.0,AnnotProjection),fn:SinceVersion(2.0,AnnotRichMedia)]	
```

Comment:
`fn:IsRequired(1.5,fn:IsPDFVersion(fn:IsPresent(State)) || fn:SinceVersion(1.6,fn:IsPresent(RT)))`
should be
`fn:IsRequired(fn:IsPDFVersion(1.5,fn:IsPresent(State)) || fn:SinceVersion(1.6,fn:IsPresent(RT)))`

Reply:
Done